lib/MusicBrainz/Server/Edit/Historic/Relationship.pm

```diff
@@ -216,7 +219,13 @@ sub _display_relationships {
             my $entity1_type = $_->{entity1_type};
             my $model0 = type_to_model( $_->{entity0_type} );
             my $model1 = type_to_model( $_->{entity1_type} );
-            my $entity0_id = $_->{entity0_id};
+            my $entity0_id;
+            if (defined $_->{entity0_id}) {
+                $entity0_id = $_->{entity0_id};
+            } else {
+                $entity0_id = $fake_entity_id;
+                $fake_entity_id++;
+            }
```

Comment:
That should be exactly equivalent to just
```suggestion
            my $entity0_id = $_->{entity0_id} // $fake_entity_id++;
```

since Perl only evaluates `$fake_entity_id++` if the left-hand side of `//` is `undef`. `$fake_entity_id++` returns the ID before the increment happens.